pep-0693.rst

```diff
@@ -48,10 +48,10 @@ Actual:
 - 3.12.0 alpha 3: Tuesday, 2022-12-06
 - 3.12.0 alpha 4: Tuesday, 2023-01-10
 - 3.12.0 alpha 5: Tuesday, 2023-02-07
+- 3.12.0 alpha 6: Wednesday, 2023-03-08
```

Comment:
```suggestion
- 3.12.0 alpha 6: Tuesday, 2023-03-07
```

It was cut on Tuesday (and released on Tuesday local time). Also, https://github.com/Yhg1s/peps/pull/1 :facepalm: (I shouldn't make PRs through unfamiliar workflows at the end of a very long day ;-P)

Reply:
Ah, I guess the date on https://www.python.org/downloads/release/python-3120a6/ is localised for the viewer, shows "March 8, 2023" here in UTC+2 👍

Reply:
It's server time, generated by the django app that we use to add releases, and server time is UTC-ish (I _think_ for historical reasons, since the server used to be in .nl? But I'm not sure.) In any case, I want to use the date it was cut to make it easier to reason about what changes might have made it into the release :)

Reply:
In that case, then, perhaps it would be best to explicitly frame this as the cut date, branching date, etc. rather than the _release_ date, which as you mention was Wednesday (UTC, of course), particularly since it may conceivably confuse users who see a different date on the releases page. But perhaps we're counting angels on the heads of pins...